artifactory.py

```diff
@@ -1478,16 +1478,31 @@ def __iter__(self):
                 continue
             yield self._make_child_relpath(name)
 
+    def read_text(self, encoding=None, errors=None):
+        """
+        Read file content
+        :param encoding: file encoding, by default Requests makes educated guesses about the encoding of
+            the response based on the HTTP headers
+        :param errors: not implemented
+        :return: (str) file content in string format
+        """
+        if errors:
+            raise NotImplementedError("Encoding errors cannot be handled")
+
+        response = self._accessor.get_response(self)
+        if encoding:
+            response.encoding = encoding
```

Comment:
We assign `encoding`, but return response.text only - how does it affect response object?

Reply:
I read how requests module is organised. Encoding should influence content of text, I added comment about it in the code